dashboards/compute/compute_network_detail.sp

```diff
@@ -136,17 +362,203 @@ query "gcp_compute_network_is_default" {
     from
       gcp_compute_network
     where
-      name = $1;
+      id = $1;
   EOQ
+}
 
-  param "name" {}
+query "compute_network_routing_mode" {
+  sql = <<-EOQ
+    select
+      'Routing Mode' as label,
+      routing_mode as value
+    from
+      gcp_compute_network
+    where
+      id = $1;
+  EOQ
 }
 
-query "gcp_compute_network_overview" {
+query "network_firewall_rules_count" {
+  sql = <<-EOQ
+    select
+      'Firewall Rules' as label,
+      count(f.*) as value,
+      case when count(f.*) > 0 then 'ok' else 'alert' end as type
+    from
+      gcp_compute_firewall f,
+      gcp_compute_network n
+    where
+      split_part(f.network, 'networks/', 2) = n.name
+      and f.project = n.project
+      and n.id = $1;
+  EOQ
+}
+
+query "auto_create_subnetwork" {
+  sql = <<-EOQ
+    select
+      'Auto Create Subnetwork' as label,
+      case when auto_create_subnetworks then 'Enabled' else 'Disabled' end as value,
+      case when auto_create_subnetworks and name = 'default' then 'ok' else 'alert' end as type
```

Comment:
Its there because GCP provides a `default` network for all projects `Auto Create Subnetwork` on, and we are just trying to verify that no manual network was created with `Auto Create Subnetwork` on.